platform/persistence/platform-persistence-commands/src/main/java/org/codice/ddf/persistence/commands/AbstractStoreCommand.java

```diff
@@ -43,15 +45,18 @@
             "--cql"}, required = false, description = "OGC CQL statement to query the persistence store. Not specifying returns all entries. More information on CQL is available at: http://docs.geoserver.org/stable/en/user/tutorials/cql/cql_tutorial.html", multiValued = false)
     protected String cql;
 
+    @Reference
+    BundleContext bundleContext;
+
     @Override
-    protected Object doExecute() {
+    public Object execute() {
 
         ServiceReference<PersistentStore> persistentStoreRef =
-                getBundleContext().getServiceReference(PersistentStore.class);
+                bundleContext.getServiceReference(PersistentStore.class);
 
         try {
             if (persistentStoreRef != null) {
-                persistentStore = getBundleContext().getService(persistentStoreRef);
+                persistentStore = bundleContext.getService(persistentStoreRef);
```

Comment:
What do `bundleContext.getService` and `bundleContext.ungetService` do? Could you use
```java
@Reference
PersistentStore persistentStore;
```
instead?